samples/test-samples.cmake

```diff
@@ -69,6 +68,11 @@ foreach (SAMPLE ${SAMPLES_LIST})
     COMMAND ${CMAKE_COMMAND} --build ${SOURCE_DIR}/${SAMPLE}
     WORKING_DIRECTORY ${SAMPLE_BUILD_DIR})
 
+  # We do not provide attestation evidence for the host-verify program, so we build, but not run, the sample.
```

Comment:
If we dont run the sample, then we cant use the sample to verify that the package installation for the host verification package is successful.

Reply:
Attestation report expires after a while that's why no sample/fake report is provided. A valid report must be generated from SGX enabled platform. It's tough to run the sample against valid report on non-SGX platform. The best we can do is ensure it builds.

Reply:
On an SGX machine we could run tools (that don't exist yet as far as I know) to generate report and cert files and then pass them to this sample. Could we track that as an open issue (doesn't need to be done in this PR)?

I think running the sample in CI is a good long term goal to avoid regressions.

Reply:
I've removed the condition after adding an oecert routine into both CMakeLists.txt and Makefile, and confirming that both ctest cases (cmake & pkg-config) can pass.